main/glossary/README.md

```diff
@@ -146,7 +139,7 @@ If they have ten houses for sale, they have ten different contract instances.
 
 ## CreatorInvitation
```

Comment:
This identifier, with the leading and midcap, does not appear anywhere else. I suggest just two words, so it's clear it is prose rather than a code identifier.
```suggestion
## Creator Invitation
```